article/app/views/fragments/articleAsideSlot.scala.html

```diff
@@ -0,0 +1,17 @@
+@(
+    shouldShowAds: Boolean,
+    optionalSlotSizes: Seq[String],
+    optClassNames: Option[String] = None
```

Comment:
I think you could replace that with a boolean `isSticky` that is `true` by default.